specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2018-09-15/DTL.json

```
@@ -10085,7 +9916,7 @@
       }
     },
     "GenerateUploadUriResponse": {
-      "description": "Response body for generating an upload URI.",
+      "description": "Reponse body for generating an upload URI.",
```

Adds back the correct spelling of "Response"
```suggestion
      "description": "Response body for generating an upload URI.",
```